docs/PerunDataGenerator.md

```diff
@@ -0,0 +1,47 @@
+# Perun Data Generator
+The perunDataGenerator.pm is a module for generating general data structure. The attributes are grouped, but
```

Comment:
Maybe mention how it can be used. I know it is not something difficult but brief information about how it needs to be imported and used may be good for someone. Perhaps also mention that it cannot be used as a separate service.